cmd/operator/operator/root.go

```diff
@@ -62,7 +62,7 @@ func NewOperatorCommand() *cobra.Command {
 	cmd.PersistentFlags().StringArrayVar(&config.SyncNamespaces, "sync-namespaces", []string{}, "namespace to watch for machine registrations")
 	_ = viper.BindPFlag("sync-namespaces", cmd.PersistentFlags().Lookup("sync-namespaces"))
 
-	cmd.PersistentFlags().StringVar(&config.OperatorImage, "operator-image", "rancher/elemental-operator:"+version.Version, "this image")
+	cmd.PersistentFlags().StringVar(&config.OperatorImage, "operator-image", "", "Operator image. Used to gather the results from the syncer by running the 'display' command")
```

Comment:
🤔 this looks to me more suited for an in-code comment (for devs) than something outputted by the help of the binary 😆

Reply:
yeah, unfortunately if we want to allow overriding this, we need a decent help for the flag :/
The idea would be to drop this somehow but I have no idea how we can not use the display command anymore to capture the logs :cry: